styles/hackage/catppuccin.user.css

```diff
@@ -242,8 +285,9 @@
     }
 
     #synopsis {
-      summary {
-        color: transparent;
+      summary,
+      #control\.syn {
+        filter: @synopsis-filter;
```

Comment:
What color is this `@synopsis-filter` supposed to be?

Reply:
`surface0` as the background, though it also changes the text similar to just inverting it so it's not ideal. I think that was why I didn't name it `@surface0-filter`.

Reply:
So is it not truly a surface0 filter then? Can you link me the page you can find this element on and maybe we can use an SVG approach or similar? I would like to move away from any nonstandard filters in #1547 now that we have https://github.com/catppuccin/userstyles/blob/main/docs/guide/images-and-svgs.md#non-svg-images-or-many-img-elements-with-external-svgs.